### PR TITLE
static content for sheep, breed texel

### PR DESCRIPTION
**Pull request by leannehaggerty**

### CODE REVIEW

File: ensembl/htdocs/ssi/species/Ovis_aries_annotation.html [hunk 1]
```diff
@@ -0,0 +1,4 @@
+</p>
+<p>The gene annotation process was carried out using a combination of protein-to-genome alignments, annotation mapping from a suitable reference species and RNA-seq alignments (where RNA-seq data with appropriate meta data were publicly available). For each candidate gene region, a selection process was applied to choose the most appropriate set of transcripts based on evolutionary distance, experimental evidence for the source data and quality of the alignments.</br>Small ncRNAs were obtained using a combination of BLAST and Infernal/RNAfold.</br> Pseudogenes were calculated by looking at genes with a large percentage of non-biological introns (introns of &lt;10bp), where the gene was covered in repeats, or where the gene was single exon and evidence of a functional multi-exon paralog was found elsewhere in the genome.</br> lincRNAs were generated via RNA-seq data where no evidence of protein homology or protein domains could be found in the transcript.</p>
```
Comment (jyothishnt) [inline ensembl/htdocs/ssi/species/Ovis_aries_annotation.html:2]: No opening <br> tag, please could you check @leannehaggerty

Comment (leannehaggerty) [reply ensembl/htdocs/ssi/species/Ovis_aries_annotation.html:1]: Sorry Jyo, think it's ok now

Comment (jyothishnt) [reply ensembl/htdocs/ssi/species/Ovis_aries_annotation.html:1]: Thanks, but also the line break should be `<br>`, not `</br`>

Comment (jyothishnt) [reply ensembl/htdocs/ssi/species/Ovis_aries_annotation.html:1]: May be you could replace `<br>`s with a just paragraphs `<p>`s?

Comment (jyothishnt) [reply ensembl/htdocs/ssi/species/Ovis_aries_annotation.html:1]:
```suggestion
<p>The gene annotation process was carried out using a combination of protein-to-genome alignments, annotation mapping from a suitable reference species and RNA-seq alignments (where RNA-seq data with appropriate meta data were publicly available). For each candidate gene region, a selection process was applied to choose the most appropriate set of transcripts based on evolutionary distance, experimental evidence for the source data and quality of the alignments.</p>

<p>Small ncRNAs were obtained using a combination of BLAST and Infernal/RNAfold</p> 

<p>Pseudogenes were calculated by looking at genes with a large percentage of non-biological introns (introns of &lt;10bp), where the gene was covered in repeats, or where the gene was single exon and evidence of a functional multi-exon paralog was found elsewhere in the genome.</p>

 <p>lincRNAs were generated via RNA-seq data where no evidence of protein homology or protein domains could be found in the transcript.</p>
```

Comment (jyothishnt) [reply ensembl/htdocs/ssi/species/Ovis_aries_annotation.html:1]: does the above suggestion look ok to you @leannehaggerty ?

Comment (leannehaggerty) [reply ensembl/htdocs/ssi/species/Ovis_aries_annotation.html:1]: Sorry I missed this Jyo, yes looks fine to me

**Review by jyothishnt — APPROVED** [review]

**Pull request merged**